jest.config.js

```diff
@@ -0,0 +1,30 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
```

Comment:
No mocks used. If somebody wants to mocks anything, he will enable the flag. Most of the time, it should not be necessary.